### DISCUSSION TITLE
Changing a transaction to transfer adds to both accounts

### DISCUSSION THREAD

**Discussion by ArthurMitchell42** [firefly-iii/firefly-iii#4674] [Firefly III]
**Bug description**
I am running Firefly III version 5.5.3, and my problem is:

After CSV import to my credit card account I correctly get transactions correctly listed including the credit card payment from my current account.
To subtract this value from the current account I tried to "convert to transfer" the CC payment
When I change the source account to the current account and the destination to the credit card acount then a positive value is added to both the credit card account (correct behaviour) and to the current account (incorrect behavior) as I would expect that this should be a debit on the current account

**Steps to reproduce**
Import transactions to the credit card account
Change manually the CC payment to a transfer

**Extra info**
Using firefox and Firefly in a Synology docker container

**Comment by JC5** [comment]
Yes, it may happen that you see a positive amount on both sides.

**Reply by ArthurMitchell42** [reply]
Is there a recomended way to handle sums from a current account that pay off a credit card then please?

**Reply by JC5** [reply]
No, it's just a visual thing.
